### ISSUE TITLE
Alt clicking a door as a AI should instantly bolt it instead of opening an UI

### ISSUE THREAD

**Issue by Graded5** [space-wizards/space-station-14#31998]
![image](https://github.com/user-attachments/assets/1f736059-a6e8-40f1-bfa6-5986d244fa0f)
It works like the title implies on SS13 last I played, and the current method is tedious.

**Comment by Dragonjspider** [comment]
Current method is like this because there will be more actions later afaik 🤷

**Comment by muburu** [comment]
there should at least be a hotkey for it since bolting might be something you wanna do quickly

**Comment by Graded5** [comment]
> Current method is like this because there will be more actions later afaik 🤷

yes but SS13 also has those actions binded to shift click and other quicker methods

**Comment by Graded5** [comment]
Just for reference on TG station:

Shift + Click = Open
Ctrl + Click = Bolt
Alt + Click = Electrify 
Ctrl + Shift + Click = Emergency Access

Since we have click for open, you could just make Shift + Click used for Emergency access

**Comment by ScarKy0** [comment]
Personally I prefer seeing a UI instead of getting shift 
 ctrl and alt as to what does what

**Comment by Graded5** [comment]
> Personally I prefer seeing a UI instead of getting shift ctrl and alt confused as to what does what

Yeah but you can always learn them, i dont see why we should limit what people willing to learn AI should be able to do. Because the UI is really slow, even if you master how to use it, its unintuitive for quick actions like door management.

**Comment by themias** [comment]
More door actions for station AI were added in #32012, so I think this can be closed.
